src/Sylius/Component/Core/Model/ProductVariant.php

```diff
@@ -345,6 +345,19 @@ public function getImages()
     /**
      * {@inheritdoc}
      */
+    public function getImage()
+    {
+        if ($this->images->isEmpty()) {
+            $image = $this->getProduct()->getImage();
+        } else {
+            $image = $this->images->first();
```

Comment:
No, I mean just remove else clause and simply return, cause if the first condition is false, it will always return it anyway. :) Like this:

``` php
if ($this->images->isEmpty()) {
    return $this->getProduct()->getImage();
}

return $this->images->first();
```